ui/insight/src/help.ts

```diff
@@ -1,14 +1,15 @@
-var m = require('mithril');
+import { h } from 'snabbdom';
+import Ctrl from './ctrl';
 
-module.exports = function (ctrl) {
-  return m('div.help.box', [
-    m('div.top', 'Definitions'),
-    m(
+export default function (ctrl: Ctrl) {
+  return h('div.help.box', [
+    h('div.top', 'Definitions'),
+    h(
       'div.content',
-      ['metric', 'dimension'].map(function (type) {
-        var data = ctrl.vm[type];
-        return m('section.' + type, [m('h3', data.name), m('p', m.trust(data.description))]);
```

Comment:
This `m.trust` exists to treat a string as html, but the description strings from the server never contain html, so the snabbdom version of the code treats descriptions as plaintext.

Reply:
> the description strings from the server never contain HTML

This seems to be true as far as I can tell but that means [description should be changed on the server](https://github.com/ornicar/lila/blob/master/modules/insight/src/main/Dimension.scala#L18) to always be a string.

Reply:
Done in https://github.com/ornicar/lila/pull/9521/commits/d8903914938633858d05dc0c96c7d97730e49888